src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -249,6 +258,22 @@ private boolean isIntegralPrimitiveType(Object incrementingColumnValue) {
            || incrementingColumnValue instanceof Byte;
   }
 
+  /**
+   * @param columnName column name
+   * @param columnAlias optional column alias
+   * @param quoteString string used to quote SQL identifiers
+   * @return the quoted column with a prefixed alias, if applicable
+   */
+  private String getQuotedColumn(String columnName, String columnAlias, String quoteString) {
+    String result = "";
+
+    if (!columnAlias.isEmpty()) {
+      result += JdbcUtils.quoteString(columnAlias, quoteString) + ".";
```

Comment:
Yeah, good point – it is the _table_ alias or name as you describe.

I would not be opposed to splitting the existing column name configuration values on period (`.`), still quoting the column name and table alias/name if applicable. Is that what you were thinking?